cfgov/data_research/tests/management/commands/test_conference_export.py

```diff
@@ -1,8 +1,13 @@
+from __future__ import unicode_literals
```

Comment:
That wasn't my goal.

test_conference_export.py tests were not running because its folder did not have an `__init__.py` file.

When I added the __init__ file, there was a no-op test and a bit of missing coverage, so I added the unicode_literal import for the new tests. I could convert them to python2 style if that's less invasive.